refnx/reduce/test/test_batch_reduce.py

```diff
@@ -169,25 +168,25 @@ def test_persistence(self, tmpdir):
 
         # presence of this file would indicate a failure of
         # the previous persistent=False
-        assert not os.path.exists(self.cache._default_persistent_cache)
+        assert not Path(self.cache._default_persistent_cache).exists()
```

Comment:
Probably should rewrite all the batch reducer in terms of pathlib too (some day)